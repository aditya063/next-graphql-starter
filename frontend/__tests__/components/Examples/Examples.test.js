import React from 'react';
import { shallow } from 'enzyme';
import Examples from '../../../components/Examples/Examples';

describe('Examples', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    mockProps = {};
    wrapper = shallow(<Examples {...mockProps} />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
