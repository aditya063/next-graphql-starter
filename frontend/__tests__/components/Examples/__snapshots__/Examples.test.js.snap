// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples matches snapshot 1`] = `ShallowWrapper {}`;
